src/shared/header/hamburgerMenu/hamburgerMenu.scss

```diff
@@ -0,0 +1,95 @@
+$hamburger-color: rgb(40, 40, 40);
+$hamburger-color-invisible: rgba(40, 40, 40, 0);
+$hamburger-color-active: rgb(240, 240, 240);
+$hamburger-menu-background: rgba(40, 40, 40, 0.9);
+$hamburger-menu-color: rgb(240, 240, 240);
+
+.hamburger {
```

Comment:
Nesting in SASS allows you to write selectors in a syntax that is more easier to read. It is one of the most basic and used feature of CSS preprocessors. It will help you use mixins i the future and support it.

Reply:
Refactored hamburger styles to make use of nesting